src/addrman.h

```diff
@@ -258,8 +258,18 @@ friend class CAddrManTest;
     //! Select several addresses at once.
     void GetAddr_(std::vector<CAddress> &vAddr, size_t max_addresses, size_t max_pct) EXCLUSIVE_LOCKS_REQUIRED(cs);
 
-    //! Mark an entry as currently-connected-to.
-    void Connected_(const CService &addr, int64_t nTime) EXCLUSIVE_LOCKS_REQUIRED(cs);
+    /** We have successfully connected to this peer. Calling this function
+     *  updates the CAddress's nTime, which is used in our IsTerrible()
```

Comment:
might be worth adding a word or two that indicates these are our primary uses of `nTime`, but not the only ones. the comment got me curious so I dove in and looked around, but its not the easiest to sift through a grep for `nTime` 😅
```suggestion
     *  updates the CAddress's nTime, which is primarily used in our IsTerrible()
```

incase curious, some other times we read from `nTime` are when processing an `ADDR` message & deciding whether to relay & for debug printing in `ConnectNode`

Reply:
> some other times we read from nTime are when processing an ADDR message & deciding whether to relay

Ah. This is using the `nTime` from a received `addr`, so not quite the same as the `nTime` stored in a `CAddress` in addrman.

> for debug printing in ConnectNode

Yes, this is true. The nTime is logged here.

I think I'm right in saying that the only _functional_ uses of nTime in a `CAddress` in addrman are in `IsTerrible()` and when relaying the `CAddress` to a peer in response to a `getaddr`.

Reply:
> I think I'm right in saying that the only functional uses of nTime in a CAddress in addrman are in IsTerrible() and when relaying the CAddress to a peer in response to a getaddr.

yup, I agree. thats why my suggestion was just to clarify that difference (esp since grep doesn't immediately make this clear). not a big deal though, maybe to consider if you touch up the code for another reason. 

> Ah. This is using the nTime from a received addr, so not quite the same as the nTime stored in a CAddress in addrman.

hm well, looks like we read the addr into a `CAddress` object and then decide whether or not to add it to our addrman? ofc the functionality of `Connected_` doesn't have the ability to impact the value at that time, but its still the same `CAddress.nTime` field thats being read?